script/integration/migrate.rb

```diff
@@ -85,6 +85,44 @@ def migrate_info_checksums(database_url)
   end
 end
 
+def migrate_created_at(database_url)
```

Comment:
But it runs after `migrate_checksums`,which populate some `create_at` and in this method you select versions with nil `created_at`. So if `migrate_checksums` is inserting wrong data, I think this won't fix? Or the data from `migrate_checksums` is legit but don't have for all versions?